charts/kube-prometheus-stack/values.yaml

```diff
@@ -2145,6 +2154,7 @@ prometheus:
       runAsUser: 1000
       fsGroup: 2000
 
+      #   - name: edns0
```

Comment:
@stafot Thanks for the contribution!

Should be a final question: Why is this added here? Or it's a leftover?

Reply:
@Xtigyro Yes it is leftover. Sorry for this. I will fix as soon as possible